src/main/java/com/redhat/sast/api/model/DataArtifact.java

```diff
@@ -2,18 +2,24 @@
 
 import java.time.LocalDateTime;
 
-import io.quarkus.hibernate.orm.panache.PanacheEntity;
 import jakarta.persistence.*;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 
 @Entity
-@Table(name = "data_artifacts")
+@Table(
+        name = "data_artifacts",
+        indexes = {@Index(name = "idx_data_artifacts_id", columnList = "id")})
 @Data
 @NoArgsConstructor
-@EqualsAndHashCode(callSuper = true)
-public class DataArtifact extends PanacheEntity {
+@EqualsAndHashCode
+public class DataArtifact {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
```

Comment:
`name` is being used when we want to provide a different name than what is defined as the variable name. By default, any schema property name defaults to the variable name. In that case, this line is unnecessary. I noticed this in other schemas as well. Please remove this.